en/docs/references/choreo-expression-syntax.md

```diff
@@ -0,0 +1,344 @@
+- `<` (less than)
+- `>` (greater than)
+- `<=` (less than or equal to)
+- `>=` (greater than or equal to)
+
+#### Examples
+- Check if the value of a variable is greater than 10
+  ```
+  myVar > 10
+  ```
+- Comparing two variables
+  ```
+  myProperty <= limit
+  ```
+
+## HTTP request related operations
+
+All the HTTP-related service applications created in Choreo have a variable called `req` (of type `http:Request`) which the developers can use to query information related to the HTTP request. Following is a list of use cases that an integration application developer would need more likely need to develop Choreo applications
+
+### Reading a header value in the request
+A typical HTTP request contains many headers. Some of these headers are standard, and some are custom. In an expression-enabled input field, you can use the `req` variables `getHeader` function to read a specific header value.
+
+#### Examples
+- Reading the value of Access-Control-Allow-Origin HTTP header
+  ```
+  req.getHeader("Access-Control-Allow-Origin")
+  ```
+- Using a value stored in a variable as the header name. Please note the omission of the double quotes.
+  ```
+  req.getHeader(headerName)
+  ```
+
+### Checking whether a header is present in the request
+
+Applications sometimes need to check the availability of a header before proceeding to something like reading the header value. You can use the  `hasHeader`  function in the req object for this purpose.
+
+#### Examples
+- Checking whether the content-length property is available as a header
+  ```
+  req.hasHeader("content-length")
+  ```
+- Using a value stored in a variable as the query parameter name. Please note the omission of the double quotes.
+  ```
+  req.hasHeader(myCustomeHeader)
+  ```
+
+### Reading the JSON Payload of the request
+
+JSON is a common content type used for HTTP communication. The req variable’s `getJsonPayload` function can be used to read the JSON payload sent with the HTTP request. Please note that the `getJsonPayload` function’s return type is `json|ClientError`.
+
+#### Examples
+- Reading the JSON Payload of the HTTP request
+  ```
+  req.getJsonPayload()
+  ```
+
+### Reading the Text Payload of the request
+
+Sometimes, content is sent as a string (plain text) with the HTTP request. The req variable’s `getTextPayload` function can be used to read the Text payload sent with the HTTP request. Please note that the `getTextPayload` function’s return type is `string|ClientError`.
+
+#### Examples
+- Reading the Text Payload of the HTTP request
+  ```
+  req.getTextPayload()
+  ```
+
+### Reading the query parameter value
+
+Query parameters are used in an HTTP request to send additional inputs to consider when processing the request. You can use the req variables `getQueryParamValue` function to read the passed query parameter.
+
+#### Examples
+- Reading the query parameter named category.
```

Comment:
We have full-stop at the end of this and not in others. Can we follow one consistent approach?

Reply:
Ack!